### PR TITLE
Implement RGB to LUV

### PR DESCRIPTION
**Pull request by ceroytres**
Adding rgb<->luv according #434 . 
Some notes:

-  Increased the numerical precision of xyz -> rgb coefficients in order to improve the precision of luv->rgb.
- It seems the opencv implementation of rgb <-> luv first converts rgb to srgb then to luv https://github.com/opencv/opencv/issues/5689
- It seems opencv doesn't apply the srgb conversion when converting to xyz which is consistent with what I implemented before https://github.com/opencv/opencv/issues/15717
- For the constants required, rgb <-> srgb <-> luv I used the constants from [skimage](https://github.com/scikit-image/scikit-image/blob/a48bf6774718c64dade4548153ae16065b595ca9/skimage/color/colorconv.py)

### CODE REVIEW

File: docs/source/color.rst [hunk 1]
```diff
@@ -21,6 +21,9 @@ The functions in this section perform various color space conversions.
 .. autofunction:: rgba_to_bgr
 .. autofunction:: rgb_to_xyz
 .. autofunction:: xyz_to_rgb
+.. autofunction:: rgb_to_luv
+.. autofunction:: luv_to_rgb
+.. autofunction:: rgb_to_luv
```
Comment (edgarriba) [inline docs/source/color.rst]: duplicated

File: kornia/color/luv.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
+from typing import Tuple
```
Comment (edgarriba) [inline kornia/color/luv.py]: endline between import gorups

File: kornia/color/luv.py [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+
+class RgbToLuv(nn.Module):
+    r"""Converts an image from RGB to Luv
+
+    The image data is assumed to be in the range of :math:`[0, 1]`. Luv
+    color is computed using the D65 illuminant and Observer 2.
+
+    args:
+        image (torch.Tensor): RGB image to be converted to Luv.
```
Comment (edgarriba) [inline kornia/color/luv.py]: make sure it outputs in the correct format, I don't see end lines

File: test/color/test_luv.py [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+                              [[0.60354551, 0.76201361, 0.79009938, 0.91742945, 0.60044175],
+                               [0.42812678, 0.18552390, 0.04186043, 0.38030245, 0.15420346],
+                               [0.13552373, 0.53955473, 0.79102736, 0.49050815, 0.75271446],
+                               [0.39861023, 0.80680277, 0.82823833, 0.54438462, 0.22063386]],
+
+                              [[0.63231256, 0.18316011, 0.84317145, 0.59529881, 0.15297393],
+                               [0.59235313, 0.36617295, 0.34600773, 0.40304737, 0.61720451],
+                               [0.46040250, 0.42006640, 0.54765106, 0.48982632, 0.13914755],
+                               [0.58402964, 0.89597990, 0.98276161, 0.25019163, 0.69285921]]]], dtype=torch.float64)
```
Comment (edgarriba) [inline test/color/test_luv.py]: it doesn't work with float32 ?

File: kornia/color/luv.py [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+
+    if not torch.is_tensor(image):
+        raise TypeError("Input type is not a torch.Tensor. Got {}".format(
+            type(image)))
+
+    if len(image.shape) < 3 or image.shape[-3] != 3:
+        raise ValueError("Input size must have a shape of (*, 3, H, W). Got {}"
+                         .format(image.shape))
+
+    # Convert from Linear RGB to sRGB
```
Comment (edgarriba) [inline kornia/color/luv.py]: not for this PR, but maybe we could have an intermediate  `rgb_to_srgb` ?

File: docs/source/color.rst (hunk 1 above)
Comment (ceroytres) [reply docs/source/color.rst]: Fixed

File: kornia/color/luv.py (hunk 2 above)
Comment (ceroytres) [reply kornia/color/luv.py]: Added

File: kornia/color/luv.py (hunk 3 above)
Comment (ceroytres) [reply kornia/color/luv.py]: Added

File: test/color/test_luv.py (hunk 4 above)
Comment (ceroytres) [reply test/color/test_luv.py]: I forgot to remove it, but the test does pass with float32.

File: kornia/color/luv.py (hunk 5 above)
Comment (ceroytres) [reply kornia/color/luv.py]: Yeah it could be useful since the opencv lab color uses the linear RGB to srgb conversion.

**Comment by edgarriba** [comment]
@ceroytres thanks for the contribution. Feel free to volunteer in any other tasks. noticed that I also updated the list of items for data augmentation

**Pull request merged**